doctrine.rst

```diff
@@ -412,6 +412,76 @@ Take a look at the previous example in more detail:
 Whether you're creating or updating objects, the workflow is always the same: Doctrine
 is smart enough to know if it should INSERT or UPDATE your entity.
 
+Validating Objects
+------------------
+
+:doc:`The Symfony validator </validation>` reuses Doctrine metadata
+to perform some basic validation tasks::
+
+    // src/Controller/ProductController.php
+    namespace App\Controller;
+
+    // ...
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+    use App\Entity\Product;
+
+    class ProductController extends AbstractController
+    {
+        /**
+         * @Route("/product", name="product")
```

Comment:
Name and route should reflect a create here otherwise one would expect an identifier to get the used object